### PR TITLE
add css3-animate-it v1.0.3 with autoupdate config

### LINKED ISSUE #7730: [Request] Add css3-animate-it
**Library name:** css3-animate-it
**Git repository url:** https://github.com/Jack-McCourt/css3-animate-it
**npm package url(optional):**
**License(s):** MIT
**Official homepage:** http://css3-animate-it.jackonthe.net
**Wanna say something? Leave message here:**

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7730 
@pvnr0082t please help me review this PR, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/Jack-McCourt/css3-animate-it
 * Official website (optional, not the repository): http://css3-animate-it.jackonthe.net/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 61
   - Count of watchers: 8
   - Count of forks: 24
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7730, cc @Jack-McCourt

### CODE REVIEW

File: ajax/libs/css3-animate-it/package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  "dependencies": {
+    "jquery": ">=1.4"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Jack-McCourt/css3-animate-it.git",
+    "basePath": "",
+    "files": [
+      "js/css3-animate-it*",
+      "css/animations*"
```
Comment (pvnr0082t) [inline ajax/libs/css3-animate-it/package.json]: @dannyxx001 I think we can use ```+(js|css)/+(css3-animate-it|animations)*```.

File: ajax/libs/css3-animate-it/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  "author": {
+    "name": "Jack McCourt",
+    "url": "https://github.com/Jack-McCourt/css3-animate-it"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Jack-McCourt/css3-animate-it.git"
+  },
+  "license": "MIT",
+  "homepage": "http://css3-animate-it.jackonthe.net/",
```
Comment (pvnr0082t) [inline ajax/libs/css3-animate-it/package.json]: @dannyxx001 This homepage show me the ```Apache2 Ubuntu Default Page```. Do you have the same situation?

**Comment by pvnr0082t** [comment]
@dannyxx001 Sorry for review late. Please rebase on the latest master branch after problems fixed. Thanks!

**Comment by dannyxx001** [comment]
@pvnr0082t Sorry for late respond. I have updated it. Please help me review again, thanks.

**Comment by pvnr0082t** [comment]
@dannyxx001 I think it's ok. Let's rebase on the latest master branch first.

**Comment by dannyxx001** [comment]
OK, I have done rebase.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**